config/src/main/java/org/springframework/security/config/web/server/OidcBackChannelServerLogoutHandler.java

```diff
@@ -137,6 +149,9 @@ String computeLogoutEndpoint(ServerHttpRequest request) {
 		int port = uriComponents.getPort();
```

Comment:
Same question as for the servlet implementation: shouldn't the server.port property be used here instead of the original request port?